live/app/routes/assessments/get-challenge.js

```diff
@@ -17,6 +17,9 @@ export default BaseRoute.extend({
       assessment: store.findRecord('assessment', assessmentId),
       challenge: store.findRecord('challenge', challengeId),
       answers: store.queryRecord('answer', { assessment: assessmentId, challenge: challengeId })
+    }).catch((err) => {
+      const meta = ('errors' in err) ? err.errors.get('firstObject').meta : null;
```

Comment:
Cool, je connaissais pas cette notation `('errors' in err)`.
Par contre, je comprend rien à ce changement et la condition qui suit.

On effectue une transition vers index si jamais il y a un champ error et que l'erreur est liée au field 'authorization' ? Si oui, on peux simplifier le code pour le rendre un peu plus explicit :)

Sinon, la transition serait plus à ça place dans le afterModel.
https://guides.emberjs.com/v2.15.0/routing/redirection/#toc_transitioning-after-the-model-is-known